jira/server/scio_search/src/main/java/ScioSearchConfigRestPlugin/impl/ScioIssueSecurityLevelScheme.java

```diff
@@ -0,0 +1,67 @@
+package ScioSearchConfigRestPlugin.impl;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.issue.security.IssueSecurityLevel;
+import com.atlassian.jira.issue.security.IssueSecurityLevelScheme;
+import com.atlassian.jira.issue.security.IssueSecuritySchemeManager;
+import com.atlassian.jira.scheme.Scheme;
+import com.atlassian.plugin.spring.scanner.annotation.imports.JiraImport;
+import com.atlassian.jira.project.Project;
+import com.atlassian.sal.api.user.UserManager;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+import java.util.ArrayList;
+
+@Named
+@Path("/issue_security_level_scheme")
```

Comment:
nit: Should we rename to `issuesecuritylevelscheme` to keep it consistent with Jira API? I think ideal would be having `project/*/issuesecuritylevelscheme` so that we can just change the base URL, not sure if it is possible though.

Reply:
I used snake case because other endpoints in the plugin were using the same. 
> project/*/issuesecuritylevelscheme

I think this is not possible.